plugins/modules/pagerduty_alert.py

```diff
@@ -53,11 +55,37 @@
             - 'triggered'
             - 'acknowledged'
             - 'resolved'
-    api_key:
+    api_version:
         type: str
         description:
-            - The pagerduty API key (readonly access), generated on the pagerduty site.
-        required: true
+            - The Api version we want to use to run the module.
+            - V1 is more limited with option we can provide to trigger incident
```

Comment:
All descriptions should contain complete sentences.
```suggestion
            - V1 is more limited with option we can provide to trigger incident.
```